app/src/main/java/com/example/ohthmhyh/CustomAdapterHF.java

```diff
@@ -85,8 +87,22 @@ public void onItemMoved(int fromPosition, int toPosition) {
     @Override
     public void onItemSwiped(int position) {
         //TODO: Add confirmation alert dialog
```

Comment:
Do you think you could an AlertDialog?

Reply:
https://github.com/CMPUT301F21T29/OneHabitTwoHabit-MyHabitYouHabit/compare/Add_alert_dialog-to-delete?expand=1

Reply:
I made a new branch just incase I broke something please dismiss this branch and goto add_alert_dialog

Reply:
Okay I will close this PR then.